dispatch/unit-tests/plistT.cc

```diff
@@ -227,4 +368,4 @@ int main(int argc, char*argv[])
 
     return wasSuccessful ? 0 : 1;
 }
-
+#endif
```

Comment:
Again just skim the code, looks all right. Does the use of unique_ptr fix the memory leak?